YogaKit/YGLayout.m

```diff
@@ -318,6 +312,14 @@ static void YGAttachNodesFromViewHierachy(UIView *const view)
   if (yoga.isLeaf) {
     YGRemoveAllChildren(node);
     YGNodeSetMeasureFunc(node, YGMeasureView);
+
+    YGSize oldSize = yoga.cachedSize;
+    YGSize newSize = YGMeasureView(node, 0, YGMeasureModeUndefined, 0, YGMeasureModeUndefined);
```

Comment:
@emilsjolander @dshahidehpour Sorry to bother you about this, but have you had time to think about my last message and the idea of introducing the feature as an "experimental" feature?